components/molecule/avatar/src/AvatarBadge/index.scss

```diff
@@ -0,0 +1,28 @@
+$s-avatar-badge: 0.6em !default;
```

Comment:
I suggest giving this one an absolute size, instead of em. 
All our brands have a different font bases, hence the size of the avatars will change from one brand to another, and I'm afraid it might lead to unexpected results.
Same with the border width of the avatar-badge-icon.

Reply:
Changed! thanks @turolopezsanabria